helm/charts/infra/templates/server/configmap.yaml

```diff
@@ -23,6 +23,8 @@ data:
 {{- end }}
 {{- end }}
 
+    version: {{ .Values.server.config.version }}
```

Comment:
Instead of templating the config with possibly an older version, it's better to template it to the current version. In this case, merging `identities` and `users` into a single list. This approach is problematic because if anyone has previously defined `identities` through helm can potentially get an unusable system because 1) `users` contains possibly admin and connector credentials and 2) `identities` overwrite `users`